packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart

```diff
@@ -383,6 +397,7 @@ abstract class GoogleMapsFlutterPlatform extends PlatformInterface {
     Set<Polygon> polygons = const <Polygon>{},
     Set<Polyline> polylines = const <Polyline>{},
     Set<Circle> circles = const <Circle>{},
+    Set<Heatmap> heatmaps = const <Heatmap>{},
```

Comment:
This is a breaking change as written; see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-platform-interface-method-parameters

https://github.com/flutter/plugins/pull/5825 will avoid the need for a breaking change here.

Reply:
https://github.com/flutter/plugins/pull/5825 has merged, is this ready to be made not a breaking change?

Reply:
Assuming I merged it correctly this should be dealt with